lib/puppet/pops/loader/task_instantiator.rb

```diff
@@ -15,7 +15,7 @@ def self.create(loader, typed_name, source_refs)
 
     files = Puppet::Module::Task.find_files(name, dirname, metadata, executables)
 
-    task = { 'name' => name, 'metadata' => metadata, 'files' => files }
+    task = { 'name' => name, 'metadata' => metadata, 'files' => files, 'module' => dirname.chomp("/tasks") }
```

Comment:
Do we need to add this here now?

Reply:
I suspect it will break puppetserver

Reply:
Michael pointed out that we can use the path from the first file in the files array - unless there's a reason not to I think we should do that and close this PR

Reply:
Although I'm not sure yet how to get the plan modulepath without making changes to Puppet, so...we may still need to make some changes here.